src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -0,0 +1,13 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    step1_label: T('SSH Keypair/Connection'),
+
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    cipher_placeholder: T('Replication Performance'),
+    cipher_tooltip: T(''),
```

Comment:
I don't know what "Replication Performance" is/does/means. Could you put some initial text here describing what it is?

Reply:
@a-stjohn This is the cipher field. We use `Replication Performance` instead of `Cipher` here.